Cache.cfc

```diff
@@ -0,0 +1,367 @@
+component extends = "lib.util.EhcacheContainer" implements = "lib.sql.IQueryable" {
+
+	Cache function init(required string name, string managerName) {
+		return super.init(argumentCollection = arguments);
+	}
+
+	any function attributesFor(required any element) {
+		local.s = createObject("java", "java.util.HashMap").init();
```

Comment:
@timmixell Is there a compelling reason for this to be a single letter variable vs something more descriptive?

Reply:
not really - i can replace it if you have a real name for it :D

Reply:
local.hashMap?